dlt/destinations/impl/filesystem/sql_client.py

```diff
@@ -74,12 +73,27 @@ def drop_authentication(self, secret_name: str = None) -> None:
         self._conn.sql(f"DROP PERSISTENT SECRET IF EXISTS {secret_name}")
 
     def create_authentication(self, persistent: bool = False, secret_name: str = None) -> None:
-        # TODO: allow users to set explicit path on filesystem where secrets are stored
-        #  https://duckdb.org/docs/configuration/secrets_manager.html#persistent-secrets
         #  home dir is a bad choice, it should be more explicit
         if not secret_name:
             secret_name = self._create_default_secret_name()
 
+        if persistent and self.memory_db:
+            raise Exception("Creating persistent secrets for in memory db is not allowed.")
+
+        current_secret_directory = self._conn.sql(
+            "SELECT current_setting('secret_directory') AS secret_directory;"
+        ).fetchone()[0]
+
+        # TODO: what is with windows?
+        is_default_secrets_directory = current_secret_directory.endswith("/.duckdb/stored_secrets")
```

Comment:
not going to work because windows use backslashes. ie. 
```
D SELECT current_setting('secret_directory') AS secret_directory;
┌──────────────────────────────────────────┐
│             secret_directory             │
│                 varchar                  │
├──────────────────────────────────────────┤
│ C:\Users\rudolfix\.duckdb\stored_secrets │
└──────────────────────────────────────────┘
```

please use Path to compare paths

Reply:
done